postgresql/db-activity-status-10.yml

```diff
@@ -0,0 +1,60 @@
+type: db
+data:
+  query: SELECT
+      datname,
+      SUM(
+        CASE
+          WHEN xact_start IS NOT NULL
+          THEN 1
+          ELSE 0
+        END
+      ) as open,
+      SUM(
+        CASE
+          WHEN state = 'idle in transaction'
+          THEN 1
+          ELSE 0
+        END
+      ) as idle_in_transaction
+    FROM pg_stat_activity
+    WHERE backend_type = 'client backend'
+    GROUP BY
+      datid,
+      datname
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+
+require:
+  - className: com.sematext.spm.client.postgresql.PostgreSQLVersionCheck
+    values: 10.0-*
+
+observation:
+  - name: activity
+    metricNamespace: postgresql
+
+    metric:
+      - name: activity.database
+        source: datname
+        type: text
+        send: false
+        label: Database
+        description: Name of the database this backend is connected to
+
+      - name: transaction.open.count
+        source: open
+        type: gauge
+        label: Active backends
+        description: Number of backends executing a query
+
+      - name: transaction.idle.count
+        source: idle_in_transaction
+        type: gauge
+        label: Idle backend
+        description: Number of backend is in a transaction, but is not currently executing a query
+
+    tag:
+      - name: activity.database
+        value: eval:${database}
```

Comment:
Does this produce anything? There should be metric named `database` which would provide value here, I don't see it? And looking into other config files, I see the same problem. But maybe I am missing something?